app/lib/efile/md/two_income_subtraction_worksheet.rb

```diff
@@ -56,22 +54,20 @@ def calculate_fed_income(primary_or_spouse)
       end
 
       def calculate_fed_subtractions(primary_or_spouse)
-        filer_json = @direct_file_json_data.filers
-          .find { |df_filer_data|
-            df_filer_data.tin.delete("-") == @intake.send(primary_or_spouse).ssn
-          }
-
-        # TODO: Some MFJ tests are missing spouse JSON - should not happen in prod
-        return 0 unless filer_json
```

Comment:
more of a question than a request for change. why is it no longer an issue if there isn't a filer_json?